### ISSUE TITLE
How was the autoencoder trained?

### ISSUE THREAD

**Issue by moha23** [mlmed/gifsplanation#1]
Hi,

The ResNet has been trained on chest xray datasets, may I get some details on how it was trained? What was the loss function used and around how many epochs was enough? Did it seem to overfit and if so how did you tackle it? (in addition, if possible kindly throw some light on how was the VQGAN trained for faceshq dataset as well?) Thanks!

**Comment by ieee8023** [comment]
Here is the training code: https://github.com/mlmed/torchxrayvision/blob/master/scripts/train_ae.py

I don't remember the number of epochs needed.
